brands/shop/car.json

```diff
@@ -324,6 +324,10 @@
     "tags": {
       "brand": "Автомир",
       "name": "Автомир",
+      "name:en": "Autoworld"
+      "shop": "car"
```

Comment:
This line is not needed as there is already a `"shop": "car"` on line 331.

Reply:
Happy to fix. How do I do it? I'm still new to github.